compiler/codegen/OMRCodeGenerator.hpp

```diff
@@ -1432,12 +1432,12 @@ class OMR_EXTENSIBLE CodeGenerator
    bool getSupportsDivCheck() {return _flags1.testAny(SupportsDivCheck);}
    void setSupportsDivCheck() {_flags1.set(SupportsDivCheck);}
 
-   bool getSupportsArrayCopy() {return _flags1.testAny(SupportsArrayCopy);}
-   void setSupportsArrayCopy() {_flags1.set(SupportsArrayCopy);}
-
    bool getSupportsPrimitiveArrayCopy() {return _flags2.testAny(SupportsPrimitiveArrayCopy);}
    void setSupportsPrimitiveArrayCopy() {_flags2.set(SupportsPrimitiveArrayCopy);}
 
+   bool getSupportsReferenceArrayCopy() {return _flags2.testAny(SupportsReferenceArrayCopy);}
```

Comment:
why is this flags2? The enum the flag is allocated in is commented as flags1 - isn't this going to test the wrong bit?